### PR TITLE
#77 sax parser

### PR DESCRIPTION
**Pull request by kaklakariada**
# Task (see #77)
Refactor parsing of specdocuments to use SAX parser instead of xml event based parsing. This allows using it under Android.

### CODE REVIEW

File: src/main/java/openfasttrack/importer/specobject/SpecObjectHandlerConfigBuilder.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    public SpecObjectHandlerConfigBuilder(final String fileName, final ImportEventListener listener)
+    {
+        this.fileName = fileName;
+        this.listener = listener;
+
+        this.handler = new CallbackContentHandler();
+    }
+
+    public EventContentHandler build()
```
Comment (tomuben) [inline src/main/java/openfasttrack/importer/specobject/SpecObjectHandlerConfigBuilder.java]: This method is quite overloaded. I suggest to refactor.

Comment (kaklakariada) [reply src/main/java/openfasttrack/importer/specobject/SpecObjectHandlerConfigBuilder.java]: Done.

**Comment by kaklakariada** [comment]
I will move the xml handling code into a sub-package in another pull request.

**Comment by kaklakariada** [comment]
Reviewed by @tomuben  and approved.

**Pull request merged**